shoop/admin/modules/products/views/forms.py

```diff
@@ -186,7 +186,20 @@ def __init__(self, **kwargs):
             self.fields["shops"].initial = [default_shop]
 
         self.file_url = self.instance.url
-        self.thumbnail = None
+
+        if self.instance.pk and self.instance.file and isinstance(self.instance.file, Image):
+            try:
+                thumbnail = self.instance.easy_thumbnails_thumbnailer.get_thumbnail({
+                    'size': (64, 64),
+                    'crop': True,
+                    'upscale': True,
+                })
+                self.file_url = self.instance.url
+                self.thumbnail = thumbnail.url
+            except Exception:
+                self.thumbnail = None
```

Comment:
The file_url is set in two places, the former one in the if is not needed.

Move the try-except to a free helper function to make this code cleaner
```python
class BaseProductMediaForm(...):
    ...
    def __init__(...):
        self.thumbnail = _try_get_thumbnail_url(self.instance)
...

def _try_get_thumbnail_url(media):
    """
    Get thumbnail URL of product media object if it is an image.

    :type media: ProductMedia  ## Is this correct?
    """
    if media.pk and isinstance(media.file, Image):
        try:
            return media.easy_thumbnails_thumbnailer.get_thumbnail(thumbnail_params)
        except WhatExactExceptionTheThumbnailerWillThrow:  # Don't use bare Exception
            pass
    return None
```